SConstruct

```diff
@@ -748,6 +756,7 @@ if selected_platform in platform_list:
             if conf.CheckCHeader(header[0]):
                 env.AppendUnique(CPPDEFINES=[header[1]])
 
+
```

Comment:
does black not fix whitespace? I have been lazy about my whitespace expecting black to format it.